scope.md

```diff
@@ -1,10 +1,10 @@
 ## Purpose and Scope
 
-The purpose of DIDComm is to provide a secure, private communication methodology built atop the decentralized design of [DIDs](https://www.w3.org/TR/did-core/).
+The purpose of DIDComm is to provide a secure, private communication methodology built atop the decentralized design of Decentralized Identities, also known as [DIDs](https://www.w3.org/TR/did-core/).
```

Comment:
Should be Identifiers instead of Identities

```suggestion
The purpose of DIDComm is to provide a secure, private communication methodology built atop the decentralized design of Decentralized Identifiers, also known as [DIDs](https://www.w3.org/TR/did-core/).
```